CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+## Contributing
+
+Like any other open source projects, there are multiple ways to contribute to this project:
+
+* As a developer, depending on your skills and experience,
+* As a user who enjoys the project and wants to help.
+
+
+##### Reporting Bugs
+
+If you found something broken or not working properly, feel free to create an issue in Github with as much information as possible, such as logs and how to reproduce the problem. Before opening the issue, make sure that:
+
+* You have read this documentation,
+* You are using the latest version of project,
+* You already searched other issues to see if your problem or request was already reported.
+
+
+##### Improving the Documentation
+
+You can improve this documentation by forking its repository, updating the content and sending a pull request.
+
+
+#### We ❤️ Pull Requests
+
+A pull request does not need to be a fix for a bug or implementing something new. Software can always be improved, legacy code removed and tests are always welcome!
+
+After you make a code change, to run:
+
+* Lint: npm run lint
+* Test: npm run test
+* Test coverage: npm run coveralls
```

Comment:
this is actually used for CI

you can do:

```suggestion 
* Test coverage: npm test -- --coverage
```
---

or add script with ```"coverage" : "jest --coverage"```

and make this

 ```suggestion
* Test coverage: npm run coverage
```